server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/DefaultMetaRepository.java

```diff
@@ -230,6 +231,7 @@ static MirrorCredential findCredential(Iterable<MirrorCredential> credentials, U
                            @JsonProperty(value = "localRepo", required = true) String localRepo,
                            @JsonProperty("localPath") @Nullable String localPath,
                            @JsonProperty(value = "remoteUri", required = true) URI remoteUri,
+                           @JsonProperty("gitIgnore") @Nullable String gitIgnore,
```

Comment:
I heard that we decided to use `String` because we will introduce a UI for mirroring in the future.
However, I think even though we provide web UI, a list of ignored files is more human-readable and it would be good for debugging.
What do you think that we allow both `String` and `List<String>`?
```java
{
    "gitIgnore": "/foo\n/bar/quz\n/qux.txt",
    "gitIgnore": [
                   "/foo", 
                   "/bar", 
                   "/quz",
                   "qux.txt" 
                  ]
}
```
We can just change the type of `gitIgnore` into `Object`. For example:
```
@JsonProperty("gitIgnore") @Nullable Object gitIgnore, ...) {
    if (gitIgnore != null) {
	if (gitIgnore instanceof Iterable) {
	    gitIgnore = ...; // join the collection with "\n"
	} else if (gitIgnore instanceof String) {
	     gitIgnore = gitIgnore;
	} else {
              throw new IllegalAguementExcepiton("gitIgnore: %s (expected: either string or array)", gitIgnore);
        }
    }

    }
}
```

Reply:
LGTM!!

Reply:
SGTM. 😄

Reply:
Would it be possible to add a test case that takes an array of Strings? 😆